src/react/features/buttons/actions.js

```diff
@@ -1,25 +1,21 @@
-import { ADD_BUTTONS, REMOVE_BUTTONS, SET_OPEN, SET_ACTIVE, SET_INACTIVE } from './actionsTypes'
+import { ADD_BUTTONS, REMOVE_BUTTONS, SET_OPEN, SET_ACTIVE } from './actionsTypes'
 
-export const addButtons = (buttons) => ({
+export const addButtons = (addedButtons) => ({
```

Comment:
For me this is unnecessary repetition: addButton(buttons) already says that those buttons are addedButtons.